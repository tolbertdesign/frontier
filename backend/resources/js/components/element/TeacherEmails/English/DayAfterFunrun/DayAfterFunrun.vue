<template>
  <p>
    To: School Families
    <br>
    When: Day After Event
    <br>
    Subject: Thank you for your support!
    <br><br>
    {{ props.event_name }} Families,
    <br><br>
    Thank you so much for supporting this year’s <strong> {{ props.event_name }}!</strong>
    The program was a huge success, and I know everyone had so much fun! There’s just one more thing left to do.
    <br><br>
    <strong>We need your help collecting all of our pledges.</strong>
    As a class, we raised <strong>${{ props.class_pledge_total }}</strong> — our goal is to collect 100% of all pledges by {{ props.due_date }}.
    <br><br>All sponsors that included their email address when making a pledge will receive an email reminder and instructions to pay for their pledge.
    Please reach out to those sponsors who did NOT include their email address to let them know they can send cash or check to you, which will then be sent to school in your child's collection envelope (coming home soon).
    <br><br>Finally, please encourage your student to thank all of their sponsors for helping our school!
    <br><br>Thank you again for your support! Let me know if you have any questions.<br>
  </p>
</template>

<script>
export default {
  name: 'DayAfterFunrun',
  status: 'prototype',
  release: '1.0.0',
  props: {
    props: {
      type: Object,
      default: () => {},
    },
  },
}
</script>
