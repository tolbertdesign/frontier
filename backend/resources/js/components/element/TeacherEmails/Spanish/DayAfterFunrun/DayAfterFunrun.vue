<template>
  <p>
    Para: Familias de Escuela
    <br>
    Cuando: Dia después del evento
    <br>Tema: Gracias por su apoyo!
    <br><br>
    <strong>{{ props.event_name }}</strong> Familias,
    <br><br>
    ¡Muchas gracias por apoyar el <strong>{{ props.event_name }}</strong> de este año!
    El programa fue un gran éxito, y sé que todos se divirtieron mucho.
    Solo queda una cosa más por hacer.
    <br><br>
    Necesitamos su ayuda para reunir todas nuestras donaciones.
    Como clase, recaudamos <strong>${{ props.class_pledge_total }}</strong> - nuestro objetivo es recaudar el 100% de todas las donaciones a más tardar en la <strong>{{ props.due_date }}</strong>.
    <br><br>Todos los patrocinadores que incluyeron su dirección de correo electrónico al hacer una donación recibirán un recordatorio por correo electrónico y instrucciones para pagar su donación.
    Comuníquese con los patrocinadores que NO incluyeron su dirección de correo electrónico para informarles que pueden enviarle dinero en efectivo o cheque, que luego se enviará a la escuela en el sobre de la colección de su hijo (pronto volverá a casa).
    <br><br>Finalmente, ¡anime a su estudiante a agradecer a todos sus patrocinadores por ayudar a nuestra escuela!
    <br><br>¡Gracias de nuevo por tu ayuda! Déjeme saber si tienes alguna pregunta.<br>
  </p>
</template>

<script>
export default {
  name: 'DayAfterFunrun',
  status: 'prototype',
  release: '1.0.0',
  props: {
    props: {
      type: Object,
      default: () => {},
    },
  },
}
</script>
