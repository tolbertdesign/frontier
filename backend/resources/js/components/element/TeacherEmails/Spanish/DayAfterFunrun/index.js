import DayAfterFunrun from './DayAfterFunrun.vue'

export default DayAfterFunrun
