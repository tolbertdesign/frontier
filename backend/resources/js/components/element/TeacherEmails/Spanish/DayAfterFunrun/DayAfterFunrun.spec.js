import { shallowMount, createLocalVue } from '@vue/test-utils'
import DayAfterFunrun from './DayAfterFunrun.vue'

const localVue = createLocalVue()

const props = {
  event_name: 'event name',
  registration_code: 'registration code',
  due_date: 'due date',
  link: 'example.com',
  class_pledge_total: '500',
}

describe('DayAfterFunrun', () => {
  it('Snapshop test for DayAfterFunrun', () => {
    const wrapper = shallowMount(DayAfterFunrun, {
      localVue,
      propsData: {
        props: props,
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
