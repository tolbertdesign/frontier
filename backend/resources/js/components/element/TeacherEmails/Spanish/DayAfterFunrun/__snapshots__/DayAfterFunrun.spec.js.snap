// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DayAfterFunrun Snapshop test for DayAfterFunrun 1`] = `
<p>
  Para: Familias de Escuela
  <br>
  Cuando: Dia después del evento
  <br>Tema: Gracias por su apoyo!
  <br><br> <strong>event name</strong> Familias,
  <br><br>
  ¡Muchas gracias por apoyar el <strong>event name</strong> de este año!
  El programa fue un gran éxito, y sé que todos se divirtieron mucho.
  Solo queda una cosa más por hacer.
  <br><br>
  Necesitamos su ayuda para reunir todas nuestras donaciones.
  Como clase, recaudamos <strong>$500</strong> - nuestro objetivo es recaudar el 100% de todas las donaciones a más tardar en la <strong>due date</strong>.
  <br><br>Todos los patrocinadores que incluyeron su dirección de correo electrónico al hacer una donación recibirán un recordatorio por correo electrónico y instrucciones para pagar su donación.
  Comuníquese con los patrocinadores que NO incluyeron su dirección de correo electrónico para informarles que pueden enviarle dinero en efectivo o cheque, que luego se enviará a la escuela en el sobre de la colección de su hijo (pronto volverá a casa).
  <br><br>Finalmente, ¡anime a su estudiante a agradecer a todos sus patrocinadores por ayudar a nuestra escuela!
  <br><br>¡Gracias de nuevo por tu ayuda! Déjeme saber si tienes alguna pregunta.<br></p>
`;
